lua/starfall/libs_sh/enum.lua

```diff
@@ -1384,4 +1384,22 @@ env.DAMAGE = {
 	["MISSILEDEFENSE"] = 2147483648
 }
 
+
+if CLIENT then
```

Comment:
All enum should be shared. You can use number values instead of the gmod enum vars if they don't exist in the server.